frontend/src/lc_theme.scss

```diff
@@ -0,0 +1,72 @@
+// Variables
+$lc_joinquil: #ffce1f;
+$lc_indigo: #003d66;
+$lc_parchment: #f1e9d2;
+$lc_floral_white: #faf8ef;
+$lc_ink: #2c2416;
+$lc_burgundy: #722f37;
+
+// Override Bootstrap defaults
+$primary: $lc_indigo;
+$secondary: $lc_joinquil;
+$link-color: $lc_burgundy;
+$light: $lc_floral_white;
+
+$body-bg: $lc_floral_white;
+$body-color: $lc_ink;
+
+// Typography
+$font-family-base: "Crimson Text", serif;
+$font-size-base: 1.2rem;
+$font-weight-normal: 500;
+$headings-font-family: "Cinzel", serif;
+$headings-font-weight: 500;
+$display-font-family: $headings-font-family;
+$display-font-weight: $headings-font-weight;
+
+// Layout
+$card-border-color: $lc_burgundy;
+$card-box-shadow: 0px 4px 6px rgba(0, 0, 0, 0.6);
+$card-bg: $lc_parchment;
+
+$navbar-light-active-color: $lc_joinquil;
+$navbar-dark-active-color: $lc_joinquil;
+
+// Special classes
+.text-burgundy {
+    color: $lc_burgundy;
+}
+
+.illuminated-initial {
+    font-size: 5rem;
+    font-weight: 700;
+    line-height: 1.5rem;
+    color: $lc_burgundy;
+    text-shadow: 1px 1px 2px rgba(0, 0, 0, 0.3);
+}
+
+.border-gold {
+    border: 2px solid $lc_joinquil;
+}
+
+.border-y-gold {
+    border-top: 2px solid $lc_joinquil;
+    border-bottom: 2px solid $lc_joinquil;
+}
+
+.border-top-gold {
+    border-top: 2px solid $lc_joinquil;
+}
+
+.border-bottom-gold {
+    border-bottom: 2px solid $lc_joinquil;
+}
```

Comment:
You should be able to get all of these styles from bootstrap's [border utilities](https://getbootstrap.com/docs/5.3/utilities/borders/#border), e.g. `border-top-gold` is equivalent to `border-top border-secondary border-2`. (Which has the advantage of not creating a combinatoric expansion of classes.)